src/components/identity/UserShows.tsx

```diff
@@ -1,10 +1,17 @@
 import React, { useContext, useEffect, useState } from 'react'
 import PageHeading from '../shared/PageHeading'
-import { AppContext } from '../../context/AppProvider';
-import { IShow } from '../shows/Show';
-import axios, { AxiosResponse } from 'axios';
-import ListShows from '../shows/ListShows';
-import { LinearProgress, TableContainer, Paper, Table, TableRow, TableCell } from '@material-ui/core';
+import { AppContext } from '../../context/AppProvider'
+import { IShow } from '../shows/Show'
```

Comment:
Types may be better in the stores directory. Doesn't matter that much though.

Reply:
oh yeah good call.  i'll move those over.